portfolio/src/main/webapp/script.js

```diff
@@ -44,8 +44,25 @@ async function getGreeting() {
 
   fetch('/data').then(response => response.json()).then((greetings) => {
     const greetingElem = document.getElementById('greeting-container');
-    var index = Math.floor(Math.random() * 10) % 3;
+    var index = Math.floor(Math.random() * greetings.length);
     greetingElem.innerHTML = greetings[index];
   });
+}
+
+// Fetch comments from server and add a random one to page.
+async function getComments() {
   
+  //const response = await fetch('/data');
+  //const greeting = await response.text();
+  //document.getElementById('greeting-container').innerHTML = greeting;
+
+  fetch('/view-comments').then(response => response.json()).then((comments) => {
+    const commentElem = document.getElementById('comment-container');
+    var i;
```

Comment:
note that since the index isn't really important, you can use a [for/of loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) here and avoid the index var